pkg/terraform/terraform.go

```diff
@@ -105,35 +105,22 @@ func SetSecurityGroupRule(rootBody *hclwrite.Body, ipAddress, sgID string, ipInT
 }
 
 // SetElasticIP attach elastic IP to our ec2 instance
-func SetElasticIP(rootBody *hclwrite.Body) {
+func SetElasticIP(rootBody *hclwrite.Body, numNodes uint32) {
```

Comment:
SetElasticIPs ?

Reply:
also update the function comment

Reply:
comment continues refering to single ec2 instance and elastic IP instead of multiple ones